src/patternfly/utilities/mixins.scss

```diff
@@ -61,6 +61,120 @@
   }
 }
 
+/// Creates flexbox fallback, with gutters for ie11 
+/// @group mixins
+/// @moduleType mixin
+/// @parameter {Suffix} xs, sm, md, lg, xl, null
+@mixin FlexboxFallback($suffix: null) {
+  // Generate column widths
+  // If suffix == default, ignore the suffix
+  $grid-unit: 100% / 12;
+
+  @for $col-num from 1 through 12 {
+    > .pf-is-#{$col-num}-col#{$suffix} {
+      flex-basis: percentage($col-num/12);
+      @supports (display: grid) {
+        flex-basis: 0;
+      }
+    }
+  }
+
+  @for $col-num from 1 through 12 {
+    > .pf-is-offset-#{$col-num}-col#{$suffix} {
+      margin-left: calc(#{$grid-unit} * #{$col-num});
+      @supports (display: grid) {
+        margin-left: 0;
+      }
+    }
+  }
+}
+
+/// Creates flexbox fallback, with gutters for ie11 
+/// @group mixins
+/// @moduleType mixin
+/// @parameter {Suffix} xs, sm, md, lg, xl, null
+@mixin FlexboxWithGuttersFallback($suffix: null) {
+  // Generate column widths
+  $grid-unit: 100% / 12;
+  &.pf-has-gutters {
+    @for $col-num from 1 through 12 {
+      $flex-basis: percentage($col-num/12);
+
+      > .pf-is-#{$col-num}-col#{$suffix} {
+        flex-basis: calc(#{$flex-basis} - #{$pf-global--gutter});
```

Comment:
Same here, we should never use a global variable on a local component or layout for the 2 tier system to work